### PR TITLE
doc: fix typos in blog/release/v10.0.0.md

### PR DESCRIPTION
**Pull request by vsemozhetbyt**

### CODE REVIEW

File: locale/en/blog/release/v10.0.0.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ The following APIs have been deprecated in Node.js 10.0.0
 * Previously deprecated internal getters/setters on `net.Server` has reached end-of-life and have been removed. [[`3701b02309`](https://github.com/nodejs/node/commit/3701b02309)]
 * Use of non-string values for `process.env` has been deprecated in documentation. [[`5826fe4e79`](https://github.com/nodejs/node/commit/5826fe4e79)]
 * Use of `process.assert()` will emit a runtime deprecation warning. [[`703e37cf3f`](https://github.com/nodejs/node/commit/703e37cf3f)]
-* Previously deprecated `NODE\_REPL\_HISTORY\_FILE` environment variable has reached end-of-life and has been removed. [[`60c9ad7979`](https://github.com/nodejs/node/commit/60c9ad7979)]
```
Comment (WaleedAshraf) [inline locale/en/blog/release/v10.0.0.md:116]: I think back-slash is required here for website.
If you check here: https://nodejs.org/en/blog/release/v10.0.0/
back-slash doesn't appear.

File: locale/en/blog/release/v10.0.0.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ The following APIs have been deprecated in Node.js 10.0.0
 * [[`876836b135`](https://github.com/nodejs/node/commit/876836b135)] - **(SEMVER-MAJOR)** **benchmark**: rename file (Ruben Bridgewater) [#18790](https://github.com/nodejs/node/pull/18790)
 * [[`e9ec9ff269`](https://github.com/nodejs/node/commit/e9ec9ff269)] - **(SEMVER-MAJOR)** **benchmark**: add buffer fill benchmark (Ruben Bridgewater) [#18790](https://github.com/nodejs/node/pull/18790)
 * [[`94d64877ff`](https://github.com/nodejs/node/commit/94d64877ff)] - **(SEMVER-MAJOR)** **benchmark**: improve buffer.readInt(B|L)E benchmarks (Rich Trott) [#11146](https://github.com/nodejs/node/pull/11146)
-* [[`9d4ab90117`](https://github.com/nodejs/node/commit/9d4ab90117)] - **(SEMVER-MAJOR)** **buffer**: do deprecation warning outside `node\_modules` (Anna Henningsen) [#19524](https://github.com/nodejs/node/pull/19524)
```
Comment (WaleedAshraf) [inline locale/en/blog/release/v10.0.0.md:141]: same here ^
I think backslash is required.

**Review by WaleedAshraf — CHANGES_REQUESTED** [review]

**Review by WaleedAshraf — APPROVED** [review]

File: locale/en/blog/release/v10.0.0.md (hunk 1 above)
Comment (vsemozhetbyt) [reply locale/en/blog/release/v10.0.0.md:116]: They are not needed if they are in the code backticks and in this case they appear (try to search with them):

bb

Comment (WaleedAshraf) [reply locale/en/blog/release/v10.0.0.md:116]: You are right. Sorry.
Updated the review. 👍

File: locale/en/blog/release/v10.0.0.md (hunk 2 above)
Comment (vsemozhetbyt) [reply locale/en/blog/release/v10.0.0.md:141]: The same:

bb2

**Review by marswong — APPROVED** [review]

**Comment by fhemberger** [comment]
Thanks!

**Pull request merged**